android-lib/src/main/java/com/kinvey/android/Client.java

```diff
@@ -1064,5 +1064,10 @@ public void stopKinveyHandlerThread() {
             kinveyHandlerThread.interrupt();
         }
     }
+
+    @Override
+    public String getDeviceId() {
+        return new UdidFactory(context).getDeviceUuid().toString();
```

Comment:
If we can, we should fix the misspelling of `UdidFactory`.

Reply:
This is internal class, we can. Updated.